core/src/test/java/org/springframework/security/provisioning/InMemoryUserDetailsManagerTests.java

```diff
@@ -37,4 +38,11 @@ public void changePassword() {
 		this.manager.updatePassword(this.user, newPassword);
 		assertThat(this.manager.loadUserByUsername(this.user.getUsername()).getPassword()).isEqualTo(newPassword);
 	}
+        
+        @Test
+	public void changePasswordWhenUsernameIsNotInLowercase() {
```

Comment:
Instead of any changes in PasswordEncodedUser, use something like this to create the user in the test method (not a member variable because it is only used once):

```java
UserDetails userNotLowerCase = User.withUserDetails(PasswordEncodedUser.user())
    .username("User")
    .build();
```

Reply:
Ok, It's done, thank you.